wordle-web/pages/dailyLeaderboard.vue

```diff
@@ -0,0 +1,53 @@
+<template>
```

Comment:
This page seems to display improperly when I run from your deployment, and when I run locally.